index.js

```diff
@@ -312,4 +312,14 @@ input MovePageInput {
   sectionId: ID!
   position: Int!
 }
+
+input CreateOtherAnswerInput {
+  id: ID!
+  type: AnswerType!
```

Comment:
Should we leave the type off for now? Since we're only dealing with `TextArea` types for the foreseeable.